src/main/java/seedu/address/model/person/Interaction.java

```diff
@@ -9,25 +11,27 @@ public class Interaction {
      * Represents the outcome of the interaction.
      */
     public static enum Outcome {
+        CLOSED,
```

Comment:
Since this is part of #65, shall we not include this in this PR?

Reply:
Rebased!